lib/driver.js

```diff
@@ -274,7 +275,8 @@ class XCUITestDriver extends BaseDriver {
       log.info(`Xcode version set to '${this.xcodeVersion.versionString}' ${tools}`);
 
       this.iosSdkVersion = await getAndCheckIosSdkVersion();
```

Comment:
Why not just use `this.opts.iosSdkVersion` and do away with `this.iosSdkVersion`.

Reply:
According to https://github.com/appium/appium-xcuitest-driver/blob/6e6eefd2381a407ac867eda2560210dd13d0a93c/lib/driver.js#L165-L176 , `this.iosSdkVersion` is imported from ios-driver. thus, I added `this.opts.iosSdkVersion` keeping `this.iosSdkVersion` to avoid affection to this.iosSdkVersion.

I replaced this.iosSdkVersion to this.opts.iosSdkVersion once and ran some tests without issues, but I thought the change should have created as another PR.

Reply:
I'm fine with that.